src/sass/modules/_m-schemaform.scss

```diff
@@ -1,5 +1,28 @@
 // Style for schemaform
 
+.va-growable-background {
+  padding: 1em;
+  margin-bottom: 1em;
+  background-color: $color-gray-lightest;
+}
+
+.progress-box .va-growable button {
+  width: auto;
+  margin-right: 0;
+  @media screen and (max-width: 340px) {
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+}
+
+.float-right {
+  float: right;
+}
+
+.float-left {
+  float: left;
+}
+
 legend {
```

Comment:
How come this had to be added?

Reply:
We had one file with these "growable" classes and the float classes and the only place they're used is in the schemaform apps, so it seemed like a good idea to consolidate them.